lib/services.js

```diff
@@ -554,14 +554,19 @@ export default {
                         return cb(errors.NoSuchUpload);
                     }
 
+                    if (!isMpuAuthorized(mpuBucket, authInfo, response,
```

Comment:
where is this whole function (metadtataValidateMultipart) still used?  Is it possible to still use this function in put part so we don't have duplication if this is still being used or should we get rid of this altogether?